latch_cli/centromere/utils.py

```diff
@@ -173,6 +176,15 @@ def _construct_ssh_client(host_ip: str, username: str):
     return ssh
 
 
+def get_default_dockerfile(pkg_root: Path, wf_type: WorkflowType):
+
```

Comment:
```suggestion
def get_default_dockerfile(pkg_root: Path, wf_type: WorkflowType) -> Path:
    """Returns the default Dockerfile at the given package root if it exists, otherwise one is generated in the `.latch` sub-directory."""
```